foundations/javascript_basics/javascript_developer_tools.md

```diff
@@ -41,7 +41,7 @@ Google has updated some of the required sections in the below tutorials and some
     -  Mobile Simulation
         1.  [Simulate mobile devices with Device Mode](https://developer.chrome.com/docs/devtools/device-mode/)
     -  JavaScript
-        1.  [Debug JavaScript](https://developer.chrome.com/docs/devtools/javascript/)
+        1.  [Debug JavaScript](https://developer.chrome.com/docs/devtools/javascript/) - Warning: In point 4 of step 3 of the tutorial, devtools will pause on the second line rather than at the declaration of the function. Don’t worry this is expected.
```

Comment:
Since "second" line isn't _totally_ clear (i.e. it's not clear to me if the declaration should be considered the first line or the 0th line) could you be a bit more specific about where it is pausing? I'm thinking probably by saying "on the `if`" but I'm not dead set on that if you have other ideas. Just saying the line number would also work.

Reply:
I completely agree with you. I think making it more specific could also help avoid unnecessary confusion. How about doing this instead?

1.  [Debug JavaScript](https://developer.chrome.com/docs/devtools/javascript/) - Warning: In point 4 of step 3 of the tutorial, devtools will pause on the second line (`if (inputsAreEmpty()) {`) rather than at the declaration of the function. Don’t worry this is expected.